creusot/tests/should_succeed/bug/570.stderr

```diff
@@ -0,0 +1,34 @@
+warning: struct `S1` is never constructed
+ --> 570.rs:4:8
+  |
+4 | struct S1 {
+  |        ^^
+  |
+  = note: `#[warn(dead_code)]` on by default
+
+warning: struct `S2` is never constructed
+ --> 570.rs:8:8
+  |
+8 | struct S2 {
```

Comment:
Can you fix these warnings by making things public?

Reply:
Done